Build/Customizations/Projects.props

```diff
@@ -9,11 +9,11 @@
   </PropertyGroup>
   <ItemGroup>
     <ReleaseProjectFiles Include="$(SolutionDirectory)\BuildScript\Buildscript.csproj" />
-    <ReleaseProjectFiles Include="$(SolutionDirectory)\ReleaseProcessScript\ReleaseProcessScript.csproj" />
+    <ReleaseProjectFiles Include="$(SolutionDirectory)\ReleaseProcsessAutomation\ReleaseProcessAutomation.csproj" />
     <UnitTestProjectFiles Include="$(SolutionDirectory)\BuildScript.UnitTests\BuildScript.UnitTests.csproj">
       <TestConfiguration>$(NormalTestConfiguration)</TestConfiguration>
     </UnitTestProjectFiles>
-    <UnitTestProjectFiles Include="$(SolutionDirectory)\ReleaseProcessScript.UnitTests\ReleaseProcessScript.UnitTests.csproj">
+    <UnitTestProjectFiles Include="$(SolutionDirectory)\ReleaseProcessAutomation.UnitTests\ReleaseProcessAutomation.UnitTests.csproj">
```

Comment:
If we replace the solutions here we should also remove the projects. Is there another task to do that? Otherwise, I think we can also remove the projects. We probably want to have that in a separate commit in the end (so one adds the new projects, and one removes the old projects and deletes them)

Reply:
Created a task for it.